src/main/kotlin/racingcar/ui/InputView.kt

```diff
@@ -1,20 +1,30 @@
 package racingcar.ui
 
-import racingcar.application.CarManager
-import racingcar.application.DefaultRandomGenerator
-import racingcar.application.RacingProcessor
-import racingcar.ui.ResultView.print
+import racingcar.application.Car
+import racingcar.application.Car.Companion.createDefault
+import racingcar.application.DefaultCarMoveConditionGenerator
+import racingcar.application.RacingGame
+import racingcar.ui.ResultView.printRacingStatus
+import racingcar.ui.ResultView.printWinners
 
 fun main() {
-    println("자동차 대수는 몇 대인가요?")
-    val numberOfCars = readlnOrNull()?.toInt() ?: throw IllegalArgumentException()
+    println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
+    val cars = inputCars()
     println("시도할 횟수는 몇 회인가요?")
     val moveTryCount = readlnOrNull()?.toInt() ?: throw IllegalArgumentException()
-
-    val cars = CarManager.createCars(numberOfCars)
-    val racingProcessor = RacingProcessor(DefaultRandomGenerator)
+    val racingGame = RacingGame(DefaultCarMoveConditionGenerator, cars)
     println("실행 결과")
     repeat(moveTryCount) {
-        print(cars = racingProcessor.race(cars))
+        racingGame.race()
+        printRacingStatus(cars)
     }
+    printWinners(cars)
+}
+
+private fun inputCars(): List<Car> {
+    val cars =
+        readlnOrNull()?.split(",")
+            ?.map { createDefault(name = it) }
+            ?: throw IllegalArgumentException()
+    return cars
```

Comment:
이 케이스는 꼭 변수 선언을하지 않아도 괜찮을 것 같네요. 😃
추가적으로 예외 발생 시, 적절한 에러 메시지를 노출해주도록 변경해주면 좋을 것 같습니다! 

```suggestion
    return readlnOrNull()?.split(",")
        ?.map { createDefault(name = it) }
        ?: throw IllegalArgumentException()
```

Reply:
네! 변수선언하는 부분 수정하도록 하겠습니다. 적절한 에러메시지 응답도요🙇‍♂️